vulkan_generator/vulkan_parser/parser.py

```diff
@@ -16,11 +16,61 @@
 
 
 from pathlib import Path
+import sys
 
 from vulkan_generator.vulkan_parser.api import types
-from vulkan_generator.vulkan_parser.internal import parser as vulkan_parser
+from vulkan_generator.vulkan_parser.internal import parser as internal_parser
 from vulkan_generator.vulkan_parser.postprocess import postprocess
 
 
-def parse(filename: Path) -> types.VulkanInfo:
-    return postprocess.process(vulkan_parser.parse(filename))
+def print_vulkan_metadata(vulkan_metadata: types.VulkanInfo) -> None:
```

Comment:
Could this just be print(vulkan_metadata)? It seems weird we need to call print on every object member individually

Reply:
True, It used to be that I was printing specific things. We don't need a separate function anymore.